app/src/main/java/org/mozilla/fenix/settings/PhoneFeature.kt

```diff
@@ -21,26 +23,17 @@ import org.mozilla.fenix.settings.sitepermissions.AUTOPLAY_BLOCK_AUDIBLE
 import org.mozilla.fenix.utils.Settings
 import android.Manifest.permission.CAMERA as CAMERA_PERMISSION
 
-const val ID_CAMERA_PERMISSION = 0
-const val ID_LOCATION_PERMISSION = 1
-const val ID_MICROPHONE_PERMISSION = 2
-const val ID_NOTIFICATION_PERMISSION = 3
-const val ID_AUTOPLAY_AUDIBLE_PERMISSION = 4
-const val ID_AUTOPLAY_INAUDIBLE_PERMISSION = 5
-
-enum class PhoneFeature(val id: Int, val androidPermissionsList: Array<String>) {
-    CAMERA(ID_CAMERA_PERMISSION, arrayOf(CAMERA_PERMISSION)),
-    LOCATION(ID_LOCATION_PERMISSION, arrayOf(ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION)),
-    MICROPHONE(ID_MICROPHONE_PERMISSION, arrayOf(RECORD_AUDIO)),
-    NOTIFICATION(ID_NOTIFICATION_PERMISSION, emptyArray()),
-    AUTOPLAY_AUDIBLE(ID_AUTOPLAY_AUDIBLE_PERMISSION, emptyArray()),
-    AUTOPLAY_INAUDIBLE(ID_AUTOPLAY_INAUDIBLE_PERMISSION, emptyArray());
+@Parcelize
+enum class PhoneFeature(val androidPermissionsList: Array<String>) : Parcelable {
+    CAMERA(arrayOf(CAMERA_PERMISSION)),
+    LOCATION(arrayOf(ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION)),
+    MICROPHONE(arrayOf(RECORD_AUDIO)),
+    NOTIFICATION(emptyArray()),
+    AUTOPLAY_AUDIBLE(emptyArray()),
+    AUTOPLAY_INAUDIBLE(emptyArray());
 
     fun isAndroidPermissionGranted(context: Context): Boolean {
-        return when (this) {
-            CAMERA, LOCATION, MICROPHONE -> context.isPermissionGranted(androidPermissionsList.asIterable())
-            NOTIFICATION, AUTOPLAY_AUDIBLE, AUTOPLAY_INAUDIBLE -> true
-        }
+        return context.isPermissionGranted(androidPermissionsList.asIterable())
```

Comment:
We used to return `true` for `NOTIFICATION`, `AUTOPLAY_AUDIBLE` and `AUTOPLAY_INAUDIBLE`, are we still returning it with the new implementation? 🤔

Reply:
Yes, because those permissions have an empty `androidPermissionsList`. Empty lists automatically return true.

Reply:
👍